### ISSUE TITLE
Game freezes up, then crashes to desktop (if fullscreen, if windowed it just indefinitely freezes)

### ISSUE THREAD

**Issue by rflcrx** [MaFi-Games/Captain-of-Industry-issues#1613]
### Issue description

Seems to happen randomly but the longer the game has been played (savefile, not play session) the more frequently it seems to happen. If I just have the game running it seems to not happen on its own but then any action (moving camera, clicking various parts of the UI) can cause the issue. The log files often seem to just cut off and not actually log everything that is happening. This only happens on my gaming laptop, not on my desktop although both devices are similiar and far above required specifications.

### Reproduction steps

1. Load the save
2. Check the log

### Game version and build number

v0.6.4d b343

### Operating system

Windows

### Attachments

[24-07-12_10-24-06_7537.log](https://github.com/user-attachments/files/16190685/24-07-12_10-24-06_7537.log)
[24-07-12_10-25-45_0124.log](https://github.com/user-attachments/files/16190686/24-07-12_10-25-45_0124.log)
[24-07-12_10-44-00_6102.log](https://github.com/user-attachments/files/16190687/24-07-12_10-44-00_6102.log)
[24-07-12_10-59-10_1834.log](https://github.com/user-attachments/files/16190688/24-07-12_10-59-10_1834.log)
[24-07-12_11-03-52_5771.log](https://github.com/user-attachments/files/16190689/24-07-12_11-03-52_5771.log)
[24-07-12_11-07-04_4189.log](https://github.com/user-attachments/files/16190691/24-07-12_11-07-04_4189.log)
[saves.zip](https://github.com/user-attachments/files/16190703/saves.zip)

**Comment by poltime** [comment]
This is due to overheating of the laptop.

**Comment by rflcrx** [comment]
> This is due to overheating of the laptop.

No. Temperatures are completely low & stable. After such a freeze I can immediately restart the game and it can remain stable for 30-60 minutes. Or it can freeze within 5 seconds again. But it's certainly not temperature related.

**Comment by poltime** [comment]
Did you do undervolting?
